filters/__init__.py

```diff
@@ -128,3 +129,20 @@ def nl2br(eval_ctx, value):
     if eval_ctx.autoescape:
         result = Markup(result)
     return result
+
+
+YOUTUBE_URL = "https://www.youtube.com/watch?v={id}&start={start}"
+YOUTUBE_EMBED_URL = "https://www.youtube-nocookie.com/embed/{id}?start={start}"
```

Comment:
`youtube-nocookie.com` -- cool! didn't know that was a thing :cookie:

Reply:
Yeah apparently it's "privacy mode", idk what that means though.